helm-charts/README.md.tpl

```diff
@@ -40,7 +40,7 @@ using:
 helm install $MY_MODEL_NAME seldonio/{{ .Name }} --namespace $MODELS_NAMESPACE
 ```
 
-{{- end -}}
```

Comment:
If `if` and `else` have no `-` why did they decided the have a trailing `-` in the `end`...

Reply:
They do magic things. I think that the `-` at the beginning removes the previous new line. And the `-` at the end, removes the following new line (both cases if there is any).